mobile/lib/shared/services/asset.service.dart

```diff
@@ -106,6 +211,128 @@ class AssetService {
     }
   }
 
+  Future<Asset> loadExif(Asset a) async {
+    a.exifInfo ??= await _db.exifInfos.get(a.id);
```

Comment:
Not quite, Asset does not load it's exifInfo (other DB table) automatically from the DB.
This line means "if exifInfo is null because it has not yet been requested, load it from the DB"